frontend/packages/gitops-plugin/integration-tests/features/gitops/add-manifest-url.feature

```diff
@@ -23,20 +21,34 @@ Feature: Applications managed by GitOps and Create secret
              Then user will see the list GitOps application groupings on the page
 
         @regression @manual
-        Scenario: Application Details page for Applications: GO-01-TC04
+        Scenario: Application Details page for Applications: GO-01-TC03
             Given user is on the Environments page
               And user can see the Applications on the page
              Then user can add new environments using kam cli
 
         @regression @manual
-        Scenario: Application Details page for Applications: GO-01-TC03
+        Scenario: Application Details page for Applications: GO-01-TC04
+            Given user is on the Environments page
+              And user can see the Applications on the page
+             Then user can see various environments for that application
+              And user can see "overview" and "Deployment history" tab for that application
+              And user can see "dev", "stage" and "prod" workloads that are deployed in the environment
+              And user can see status of the workloads deployed in the environment
+              And user can see "Last deployed" date for workloads that are deployed in the environment
+              And user can see a clickable "Delpoyment history" for each workload that are deployed in the environment
```

Comment:
Remove this line, see above comment about deployment history this release

Reply:
Done